packages/fs-detectors/test/fixtures/42-npm-workspace-with-nx/workspace.json

```diff
@@ -0,0 +1,4 @@
+{
+  "version": 2,
+  "projects": {}
```

Comment:
is this a valid scenario to have an empty workspace.json?

Reply:
Yes it is. It is when you initialize the project with `npx create-nx-workspace --package-manager=npm`